addon/components/justa-table.js

```diff
@@ -22,6 +22,20 @@ export default Component.extend({
     }
   },
 
+  didInsertElement() {
+    this._super();
+    Ember.run.scheduleOnce('afterRender', this, this.afterRenderEvent);
```

Comment:
Seems like only doing this in `didInsertElement` will give problems when columns are added/removed. We should probably use `didRender` (https://guides.emberjs.com/v2.3.0/components/the-component-lifecycle/#toc_making-updates-to-the-rendered-dom-with-code-didrender-code) instead. something like this:

```js
{
  didRender() {
    this._super(...arguments);
    this._uninstallStickyHeaders();
    this._installStickyHeaders();
  },

  willDestroyElement() {
    this._uninstallStickyHeaders()
  },

  _installStickyHeaders() {
    if (this.get('stickyHeader')) {
      this.$('table').stickyTableHeaders();
    }
  },

  _uninstallStickyHeaders() {
     if (this.get('stickyHeader')) {
      this.$('table').stickyTableHeaders('destroy');
    }
  }
}
```

Also note the added destroy call and using `this.$` to scope to the component's element.